docker/terra-node/entrypoint/test.sh

```diff
@@ -0,0 +1,5 @@
+#/bin/sh -e
+
+sh /opt/entrypoint/npm_install.sh
```

Comment:
Would the npm_intall script be called when the image was build?

Reply:
Npm install needs to be run in a container either opts out of running npm install (env npm_install=false) during the build phase or when a volume is mounted that overwrites node modules/lib. In the case of running tests i'd assume the latter where it can allow a dev to short circuit having to build then run. They can make test changes and just run  ```docker-compose run test```. With the volume mount too will allow for caching of node modules and the us of this npm instlal script will not re-install node modules if they are already installed.

Reply:
hmm gottchya. Makes sense. Not sure if it would be worth a short blurb here since most people building docker containers wouldn't be copying local node_modules into the container?? I have no strong opinion. 👍